src/components/notice/index.tsx

```diff
@@ -0,0 +1,33 @@
+// tslint:disable-next-line: no-implicit-dependencies
+import { Component, Prop, h, Listen } from "@stencil/core"
+import { Trigger, Notice } from "smoothly-model"
+
+@Component({
+	tag: "smoothly-notice",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyNotice {
+	@Prop() notice: string | Notice = { type: "warning", message: "An error occured."}
+	@Prop({ mutable: true }) show: boolean = false
```

Comment:
Show in this usage is a verb. Properties should normally be adjectives or substantives. (Methods verbs and classes substantives.) Use the name `visible` instead.

Reply:
Or just let it trigger on the presence of a notice.